src/styles/blocks/star.scss

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    @include star($star-size);
+
+    list-style: none;
+    margin-right: 4px;
+
+    &:last-child {
+      background: url(./star.svg) no-repeat center;
+    }
+  }
```

Comment:
Your stars block should be flexible for all modificators, why do you remove them?